.github/workflows/tests.yml

```
@@ -139,27 +145,81 @@
         name: codecov-umbrella
 
+  # Playwright E2E Tests
+  playwright-tests:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    needs: changes
+    if: needs.changes.outputs.should-run-tests == 'true'
+
+    steps:
+    
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)